### ISSUE TITLE
Compatibility with Mongodb 3.4

### ISSUE THREAD

**Issue by corporatepiyush** [michaelklishin/monger#151]
Is it compatible with MongoDB 3.4 and all new features introduced in that release ?

**Comment by xiocode** [comment]
any update?

**Comment by michaelklishin** [comment]
This is an open source project. If something isn't supported, feel free to contribute it.

**Comment by Linicks** [comment]
All/ @michaelklishin  
  I'm trying to use Monger with the MongoDB Atlas service running MongoDB 3.4.x, which requires a Java driver version of 3.4.x+( http://mongodb.github.io/mongo-java-driver/ ).  I manually added the new driver dependency to my project.clj with the following:  [org.mongodb/mongo-java-driver "3.4.2"] just before the Monger dependency definition.  This seems to be working so far, and the release notes only mention a small compatibility change here, ( http://mongodb.github.io/mongo-java-driver/3.4/upgrading/ ).  I'm new to Clojure, and haven't done Java for years, so I did't want to make a naive pull request with just a driver version change.   

Thanks!  
--Nick
